File: Java/Java source-code/manhattanDistance.java


public class manhattanDistance {
    public static double manhattanDistance(Double[] p1, Double[] p2) {
        double result = 0.0;
        for (int i = 0; i < p1.length; i++) {
            result += Math.abs(p2[i] - p1[i]);
        }
        return result;
    }
}

File: Methods-C++/manhattanDistance.cpp
#include <iostream>

double manhattanDistance(double p1[], double p2[], int n) {
    double result = 0.0;
    for (int i = 0; i < n; i++) {
        result += abs(p2[i] - p1[i]);
    }
    return result;
}
